lib/components/Form/AutoComplete/index.jsx

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const AutoComplete = ({ items, onClick }) => (
+const AutoComplete = ({ items, onClick, searchValue }) => (
   <ul className="autocomplete">
     {
-      // eslint-disable-next-line
-      items.map(item => <li onClick={() => onClick(item)}>{item.text}</li>)
+      items.map((item) => {
+        const formattedText = { __html: item.text.replace(searchValue, `<b>${searchValue}</b>`) };
+        // eslint-disable-next-line
+        return <li onClick={() => onClick(item)} dangerouslySetInnerHTML={formattedText} />
```

Comment:
What is "dangerouslySetInnerHTML"?

Reply:
https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml